utils/hct/hctbuild.cmd

```diff
@@ -204,6 +204,13 @@ if "%1"=="-default-adapter" (
   shift /1
   shift /1 & goto :parse_args
 )
+if "%1"=="-sanitizer" (
+  set CMAKE_OPTS=%CMAKE_OPTS% -DLLVM_ENABLE_SANITIZER=Address
+  set CMAKE_OPTS=%CMAKE_OPTS% -DLLVM_USE_SANITIZER:BOOL=On
```

Comment:
I don't see `LLVM_ENABLE_SANITIZER` used anywhere. Additionally, it looks as though `LLVM_USE_SANITIZER` is expected to be a string not a bool.

Reply:
Fixed.